parser_library/src/diagnostic.cpp

```diff
@@ -1857,6 +1864,13 @@ diagnostic_op diagnostic_op::error_ME002(const range& range)
     return diagnostic_op(diagnostic_severity::error, "ME002", "multiplication or division of address", range);
 }
 
+diagnostic_op diagnostic_op::error_ME003(const range& range)
+{
+    return diagnostic_op(diagnostic_severity::error,
+        "ME003",
+        "Relative Immediate Operand value must be signed number of halfwords.",
```

Comment:
This message implies to me, that the user should write a number as the argument.
```suggestion
        "Relative immediate operand must evaluate into a number of halfwords (must result in even number of bytes)",
```

Reply:
maybe just `Relative immediate operand must evaluate into an even offset`?